Tools/scripts/build_options.py

```diff
@@ -178,6 +178,7 @@ def __init__(self,
     Feature('Other', 'NMEA_OUTPUT', 'HAL_NMEA_OUTPUT_ENABLED', 'Enable NMEA Output', 0, None),
     Feature('Other', 'BARO_WIND_COMP', 'HAL_BARO_WIND_COMP_ENABLED', 'Enable Baro Wind Compensation', 0, None),
 
+    Feature('GPS Drivers', 'Ublox', 'AP_GPS_UBLOX_ENABLED', 'Enable Ublox GPS', 1, 'None'),
```

Comment:
```suggestion
    Feature('GPS Drivers', 'UBLOX', 'AP_GPS_UBLOX_ENABLED', 'Enable u-blox GPS', 1, None),
```

(that's apparently they way they branch their GPS, caps-wise...)